hetzner/ansible/hetzner-create-hcloud-server.yml

```diff
@@ -62,7 +62,7 @@
       failed_when: "hcloud_delete_key_res.rc != 0 and ('SSH key not found' not in hcloud_delete_key_res.stderr)"
       when: "vm_delete is defined and vm_delete | bool"
 
-    - name: Create cloud server key
+    - name: "Create cloud server key - TODO: Use passwordstore"
```

Comment:
Use we use the `password store`, will the playbook git clone the project ?

Reply:
IMO we should leave the source of the `password store` project out of the scope of this playbook.

Reply:
Then, you must document that a pre-req will be to have the `password-store` project cloned locally